### PR TITLE
Create awair.json

### PR DESCRIPTION
**Pull request by deanlyoung**
First commit

### CODE REVIEW

File: data/apps/awair.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
+{
+  "info": {
+    "identifier": "awair",
+    "name": "Awair - Breathe Easy",
```
Comment (pietropizzi) [inline data/apps/awair.json]: Could you just have the name here instead of with the tagline?

File: data/apps/awair.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    "appstoreUrl": "https://itunes.apple.com/app/id1034660006"
+  },
+  "actions": [
+    {
+      "name": "list",
```
Comment (pietropizzi) [inline data/apps/awair.json]: I didn't really create a style guide for the documentation or anything, but it'd be really cool if the names could be in title case. And maybe the description could be a bit more descriptive like "Opens the device list view".

File: data/apps/awair.json (hunk 1 above)
Comment (pietropizzi) [reply data/apps/awair.json]: It’s just that we’re the titles are used space is quite limited.

**Comment by pietropizzi** [comment]
@deanlyoung is there anything I can help you with to get this ready?

**Comment by deanlyoung** [comment]
@pietropizzi sorry! I was on vacation. I’ll get to this as soon as I can.

**Comment by pietropizzi** [comment]
> @pietropizzi I didn't see an example of where to put the app icon

You can put the icon inside of `src/images/app-icons` with the same name as the identifier of the app. So `awair.png`.

The documentation for it is quite hidden in here: https://github.com/pietropizzi/app-talk/blob/master/CONTRIBUTING.md#good-to-know

Thanks for the formatting fixes!

**Comment by pietropizzi** [comment]
Cool! Thanks!

**Pull request merged**